frontend/alert/src/components/shared/accounts/AccountIndicator.js

```diff
@@ -18,11 +18,13 @@ const AccountIndicator = ({
         fetch("/accounts/me/")
             .then((response) => {
                 if (!response.ok) {
+                    login(null);
                     return;
                 }
                 response.json()
                     .then(newUser => login(newUser));
-            });
+            })
+            .catch(() => login(null));
     }, [login]);
```

Comment:
Looks like this fetch is now getting called in a loop when i run the code –– not sure why since the dependencies for useEffect didn't change.

Reply:
fixed!